### ISSUE TITLE
Search Issues

### ISSUE THREAD

**Issue by obashistu** [thegeeklab/hugo-geekdoc#31]
Search doesn't appear to work and gives a console error (below).

Browser details:

https://www.whatsmybrowser.org/b/9XT1H

Tested on the getting started site.

![image](https://user-images.githubusercontent.com/37293383/100879083-e833fe80-34a2-11eb-81ed-0b21a12d371d.png)

**Comment by xoxys** [comment]
Can you test another browser?

**Comment by obashistu** [comment]
adding 

```
<script defer src="{{ index .Site.Data.assets "js/flexsearch.min.js" | relURL }}"></script>

```

into partials/foot.html fixed the issues (tested on FF/Chrome and Edge)

```
{{ if default true .Site.Params.GeekdocSearch }}
{{ .Scratch.Set "geekdocSearchConfig" .Site.Params.GeekdocSearchConfig }}
<!-- Remove after https://github.com/gohugoio/hugo/issues/6331 -->
{{ $searchJSFile := printf "js/%s.search.js" .Language.Lang }}
{{ $searchJS := resources.Get "js/search.js" | resources.ExecuteAsTemplate $searchJSFile . | resources.Minify | fingerprint }}
<script defer src="{{ index .Site.Data.assets "js/flexsearch.min.js" | relURL }}"></script>
<script defer src="{{ $searchJS.RelPermalink }}"></script>
{{ end }}

{{ if default true .Site.Params.GeekdocAnchorCopy }}
<script defer src="{{ index .Site.Data.assets "js/clipboard.min.js" | relURL }}"></script>
<script>
    document.addEventListener("DOMContentLoaded", function(event) {
        var clipboard = new ClipboardJS('.clip');
    });
</script>
{{ end }}
```

**Comment by xoxys** [comment]
Thanks! You're right there is an issue loading the hashed flexsearch source file. I've prepared a PR and will tag a new release later.

**Comment by xoxys** [comment]
Fixed in v0.7.3 please test. Thanks again for reporting.
